esp8266-fastled-webserver/esp8266-fastled-webserver.ino

```diff
@@ -68,7 +70,8 @@ uint8_t autoplayDuration = 10;
 unsigned long autoPlayTimeout = 0;
 
 uint8_t showClock = 0;
-uint8_t clockBackgroundFade = 240;
+uint8_t clockBackgroundFade = 160;
```

Comment:
👍 I wholeheartedly agree ... the background patterns were too faint otherwise.  👍